pipeline_noted.ipynb

```diff
@@ -119,20 +134,31 @@
    },
    "outputs": [],
    "source": [
+    "#Set up Initial Basic Parameters#\n",
     "minian_path = \".\"\n",
     "dpath = \"./demo_movies\"\n",
     "chunks = {\"frame\": 1000, \"height\": 50, \"width\": 50, \"unit_id\": 100}\n",
-    "subset = None\n",
-    "subset_mc = None\n",
     "in_memory = True\n",
     "interactive = True\n",
     "output_size = 60\n",
+    "param_save_minian = {\n",
+    "    'dpath': dpath,\n",
+    "    'fname': 'minian',\n",
+    "    'backend': 'zarr',\n",
+    "    'meta_dict': dict(seesion_id=-1, session=-2, animal=-3),\n",
+    "    'overwrite': True}\n",
+    "\n",
+    "#Pre-processing Parameters#\n",
     "param_load_videos = {\n",
     "    'pattern': 'msCam[0-9]+\\.avi$',\n",
     "    'dtype': np.float32,\n",
     "    'in_memory': in_memory,\n",
     "    'downsample': dict(frame=2),\n",
     "    'downsample_strategy': 'subset'}\n",
+    "subset = {\n",
```

Comment:
this assumes too much. It won't work, for example, if the coordinates doesn't start from 0, (like when it's reversed)